runtime/battery-station/src/lib.rs

```diff
@@ -102,7 +102,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("zeitgeist"),
     impl_name: create_runtime_str!("zeitgeist"),
     authoring_version: 1,
-    spec_version: 51,
+    spec_version: 52,
     impl_version: 1,
     apis: RUNTIME_API_VERSIONS,
     transaction_version: 26,
```

Comment:
Double-check: Sure we don't have anything that changes the `transaction_version`?